tests/Elastic.Channels.Tests/BehaviorTests.cs

```diff
@@ -53,7 +53,7 @@ [Fact] public async Task RespectsPagination()
 	/// we don't want this data equally distributed over multiple calls to export the data.
 	/// Instead we want the smaller buffer to go out over a single export to the external system
 	/// </summary>
-	[Fact] public async Task MessagesAreSequentiallyDistributedOverWorkers()
+	[Fact(Skip ="Temporary skip this on CI")] public async Task MessagesAreSequentiallyDistributedOverWorkers()
```

Comment:
Not blocking:

I just found https://stackoverflow.com/a/35871507/5658224 and was wondering about your thoughts.

Reply:
On other projects we have `SkipOnCi` attributes. 

Ideally nothing should be skipped on CI :D 

The new xunit3 (which we use on the docs) has conditionals for `Skip` through `SkipWhen()` which seems like a cleaner solution.